pyproject.toml

```diff
@@ -0,0 +1,101 @@
+[project]
+name = "rtc-tools-heat-network"
+dynamic = ["version"]
+authors = [{ name = "Jim Rojer", email = "jim.rojer@tno.nl" }]
+description = "RTC Tools Heat Network is used for ESDLS containing a heat network. It can optimize and low-fidelity simulate OPEX and CAPEX costs as well as provide the optimal asset placements."
+classifiers = [
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Operating System :: OS Independent",
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Information Technology",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Topic :: Scientific/Engineering :: Physics",
+]
+
+dependencies = [
+    "influxdb >= 5.3.1",
+    "pyecore",
+    "pymoca >= 0.9.0",
+    "rtc-tools == 2.6.0a3",
+    "pyesdl >= 21.11.0",
+    "pandas >= 1.3.1, < 2.0",
+]
+
+[project.optional-dependencies]
+dev = [
+    #"pip-tools~=6.13.0",  # Installed via create_env script
+    "black ~= 22.1.0",
+    "flake8 ~= 6.0.0",
+    "Flake8-pyproject ~= 1.2.3",
+    # "flake8-docstrings ~= 1.7.0", TODO Add in when docstrings are being added
+    "flake8-quotes ~= 3.3.2",
+    "pytest ~= 7.3.1",
+    "pytest-cov ~= 4.0.0",
+    "bump2version ~= 1.0.1",
+    "mypy ~= 1.5.1",
+    "build ~= 1.0.3",
+    "setuptools_git_versioning < 2"
+]
+
+[project.urls]
+homepage = "https://www.nwn.nu"
+documentation = "https://readthedocs.org"
+repository = "https://github.com/Nieuwe-Warmte-Nu/rtc-tools-heat-network"
+changelog = "https://github.com/Nieuwe-Warmte-Nu/rtc-tools-heat-network/blob/main/CHANGELOG.md"
+
+[project.scripts]
+newchess = "rtctools_heat_network.infrastructure.app:cmdline_app"
+
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools ~= 67.7.2", "wheel ~= 0.40.0", "setuptools-git-versioning"]
+
+[tools.setuptools]
+packages = ["src/rtctools_heat_network"]
+
+[tool.setuptools-git-versioning]
+enabled = true
+
+[tool.pytest.ini_options]
+addopts = "--cov=rtctools_heat_network --cov-report html --cov-report term-missing --cov-fail-under 67"
```

Comment:
You can change minimum % test coverage on this line. Currently set to 67 (current coverage) to make sure changes in test do not lower test coverage.

Reply:
based on the line of code that I see I do not understand what run and for what directories and/or files it checks the coverage.